pandas/core/frame.py

```diff
@@ -556,6 +556,10 @@ def __init__(
 
         NDFrame.__init__(self, mgr)
 
+    # ----------------------------------------------------------------------
+    # Array interface
+    _HANDLED_TYPES = (Series, Index, ExtensionArray, np.ndarray)
```

Comment:
can this go up with at the top with the other declarative constants like _typ (or if theres another section this makes sense as a part of)